gcp/opsverse-gke/vars.tfvars

```diff
@@ -1,10 +1,20 @@
-gcp_project_id      = "opsverse-dev-poc"
-gcp_region          = "us-east4"
-cluster_name        = "opsverse-gke-cluster"
+gcp_project_id = "opsverse-dev-poc"
+gcp_region     = "us-east4"
+cluster_name   = "opsverse-gke-cluster"
+# The bucket name should be of the format devopsnow-loki-<customer_name>-<stack_name>
```

Comment:
All this information has to be added to `README.md`. Customers refer Readme file. Let's add all these details.

Reply:
Let's highlight all this information in the README using `Important` tag.

Reply:
Done